virttest/utils_test/libvirt.py

```diff
@@ -2855,3 +2855,35 @@ def get_vol_list(pool_name, vol_check=True, timeout=5):
             raise exceptions.TestError("Volume path %s not exist" % vol_path)
 
     return vols
+
+
+def get_iothreadsinfo(vm_name, options=None):
+    """
+    Parse domain iothreadinfo.
+
+    :param vm_name: Domain name
+    :return: The dict of domain iothreads
+
+    ::
+        # virsh iothreadinfo vm2
+        IOThread ID CPU Affinity
+        ---------------------------------------------------
+        2 3
+        1 0-4
+        4 0-7
+        3 0-7
+
+    The function return a dict like:
+
+    ::
+        {'2': '3', '1': '0-4', '4': '0-7', '3': '0-7'}
+    """
+    ret = virsh.iothreadinfo(vm_name, options, ignore_status=True)
+    if ret.exit_status:
+        raise exceptions.TestFail(ret.stderr.strip())
```

Comment:
As the 'ignore_status' is True, maybe you don't need raise exception here(at least it's not a TestFail exception), and just return {}